.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
we're adding this file, but ignoring it in .gitignore. Probably need to remove it here.